Vizzuality/heco-invest
# Let 1138 fe update the sign up flow to account for email verification

This PR add the email confirmation to the sign up flow

## Testing instructions

1. Sign out;
2. Go to sign-up
3. Fill the form and click 'sign up'
4. You should be redirected to the verification page and receive the confirmation email
5. Open you email and click on the confirmation link
6. You should go to signup page with a confirmation_token query 

- If the token is invalid, an error message should appear

7. You should be redirected to the 'Choose account' page 


## Tracking

[1138](https://vizzuality.atlassian.net/browse/LET-1138)
